threestudio/systems/imagedreamfusion.py

```diff
@@ -66,22 +67,22 @@ def training_substep(self, batch, batch_idx, guidance: str):
             bg_color = None
         elif guidance == "guidance":
             batch = batch["random_camera"]
-            # claforte: surely there's a cleaner way to get batch size
             bs = batch["rays_o"].shape[0]
 
-            bg_color = torch.rand(bs, 3).to(self.device)  # claforte: use dtype
+            bg_color = torch.ones(bs, 3).to(self.device)
+            # bg_color = torch.rand(bs, 3).to(self.device)  # claforte: use dtype
 
-            # Override 50% of the bgcolors with white.
-            # This results in better predictions with Zero123,
-            # since it was trained with a constant white background.
-            white = torch.ones(bs, 3).to(self.device)
+            # # Override 50% of the bgcolors with white.
+            # # This results in better predictions with Zero123,
+            # # since it was trained with a constant white background.
+            # white = torch.ones(bs, 3).to(self.device)
 
-            # is the batch item white? shaped [bs, 1]
-            is_white = (torch.rand(bs) > 0.5).to(self.device).float().unsqueeze(-1)
+            # # is the batch item white? shaped [bs, 1]
+            # is_white = (torch.rand(bs) > 0.5).to(self.device).float().unsqueeze(-1)
 
-            bg_color = bg_color * (1.0 - is_white) + white * is_white
+            # bg_color = bg_color * (1.0 - is_white) + white * is_white
 
-            ambient_ratio = 0.1 + 0.9 * random.random()
+            ambient_ratio = 0.5 + 0.5 * random.random()
```

Comment:
We should make this configurable and leave the default at `0.1 + 0.9 * random`... otherwise you might break some others' projects

Reply:
Sure, I'll revert it back to 0.1 + 0.9*rand for now.
I didn't see any noticeable difference between the two cases though.